frontend/types/type.ts

```diff
@@ -0,0 +1 @@
+export type Color = "red" | "indigo" | "pink" | "cyan" | "blue" | "emerald" | "amber" | "gray" | "slate" | "zinc" | "neutral" | "stone" | "orange" | "yellow" | "lime" | "green" | "teal" | "sky" | "violet" | "purple" | "fuchsia" | "rose"
```

Comment:
Use the colors out of the tailwind definition or find another solution to pass colors to props.
```suggestion
import colors from "tailwindcss/colors"
export type Color = keyof typeof colors
```

Das Problem ist das jetzt transparent, white undso auch dabei ist. Sollen wir die dann noch rausnehemen mit typescript pick oderso?